HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,146 @@
----
-title: "What went wrong?"
-author: "Robert Gruener"
-date due: "7/13/2020"
-output: github_document
----
-
-```{r setup, include=TRUE}
-knitr::opts_chunk$set(echo = TRUE, error = TRUE)
-```
-
-## HW02 Part A
-
-In this document, I will add some examples of some coding mistakes, it is up to you to figure out why the graphs are messing up. 
-
-### First load packages
-
-It is always best to load the packages you need at the top of a script. It's another common coding formatting standard (like using the assignment operator instead of the equals sign). In this case, it helps people realize what they need to install for the script and gives an idea of what functions will be called. 
-
-It is also best coding practice to only call the packages you use, so if you use a package but end up tossing the code you use for it, then make sure to remove loading it in the first place. For example, I could use `library("tidyverse")` but since this script will only be using ggplot2, I only load ggplot2.
-
-```{r load libraries}
-library("ggplot2")
-library("magrittr") #so I can do some piping
-```
-
-
-### Graph Fail 1
-
-What error is being thrown? How do you correct it? (hint, the error message tells you)
-
-```{r}
-data(mpg) #this is a dataset from the ggplot2 package
-
-mpg %>% 
-  ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 

```

Comment:
Hi Monty, this is what I have for fail 1: 
mpg %>% 
  ggplot(mapping = aes(x = "city", y = "hwy", color = "blue")) + 
  geom_point()

I ran your code, but it gave me error message
"Error: `mapping` must be created by `aes()` Did you use %>% instead of +?"

Does the code ran fine on your computer?